src/core/Akka/Akka.csproj

```diff
@@ -19,6 +19,7 @@
     <PackageReference Include="System.Collections.Immutable" Version="5.0.0" />
     <PackageReference Include="System.Reflection.Emit" Version="4.7.0" />
       <PackageReference Include="System.Collections.Immutable" Version="5.0.0" />
+      <PackageReference Include="System.Threading.Channels" Version="6.0.0" />
```

Comment:
We probably need to align all of our `System.*` packages on a common version range - see https://github.com/akkadotnet/akka.net/pull/5365#issuecomment-984763805

Separate issue from this PR, but I'm seeing a trend here...

Reply:
done